### PR TITLE
Update .pullapprove.yml

### PR DESCRIPTION
**Pull request by dzearing**

### CODE REVIEW

**Comment by battletoilet** [comment]
Approved

**Comment by wdo3650** [comment]
Approved

**Comment by mikewheaton** [comment]
Approved

File: .pullapprove.yml [hunk 1]
```diff
@@ -9,5 +9,7 @@ reviewers:
   - battletoilet
   - wdo3650
   - mikewheaton
+  - dzearing
+  - cliffkoh
   name: pullapprove
-  required: 2
+  required: 1
```
Comment (mikewheaton) [inline .pullapprove.yml]: Why only 1 reviewer? We often have issues that aren't caught by the first reviewer.

**Pull request merged**